### PR TITLE
update initial confings

### PR DESCRIPTION
**Pull request by HerbertYiga**
updating initial confings

### CODE REVIEW

**Comment by HerbertYiga** [comment]
cc @pmanko

File: configs/openhim-core/initial-config.json [hunk 1]
```diff
@@ -56,6 +56,36 @@
...
+      "roles": [
+        "emr"
+      ],
+      "clientID": "isanteplus.sedish-haiti.org",
+      "name": "isanteplus.sedish-haiti.org",
+      "passwordAlgorithm": "sha512",
+      "passwordSalt": "228b11da37d15ac9971e8ec7bee28752",
+      "passwordHash": "1c8f12ebe71f7394f12c7aacc69cbe8c1ba041351e28a67364b079eb523f00368a97941b30fa960c12943bfd044d9253d79a0464b225679f8da01dc90addb54a"
+    },
+    {
```
Comment (pmanko) [inline configs/openhim-core/initial-config.json:80]: We also need the lnsp role, no?

**Review by pmanko — COMMENTED** [review]
@HerbertYiga I believe we still need an LNSP role, and have it added to the appropriate channels. 

Also, there seems to be some issue booting up OpenCR in CI: https://github.com/I-TECH-UW/sedish-haiti.org/actions/runs/4376396921/jobs/7658446596

Can you check if everything boots up locally?

**Comment by HerbertYiga** [comment]
> believe we still need an LNSP role, and have it added to the appropriate channels.
> 
> Also, there seems to be some issue booting up OpenCR in CI: https://github.com/I-TECH-UW/sedish-haiti.org/actions/runs/4376396921/jobs/7658446596
> 
> Can you check if everything boots up locally?

okie let me double check this!

**Comment by HerbertYiga** [comment]
hi @pmanko  so i notice that things dont boot up locally from the main branch, i get such an error( Couldn't find env file: /home/christ/projects/uwash/haiti/sedish-haiti.org/configs/opencr/.env)  
We might need to merge all the current changes to master at the  server

File: configs/openhim-core/initial-config.json [hunk 2]
```diff
@@ -56,6 +56,46 @@
...
+      "passwordAlgorithm": "sha512",
+      "passwordSalt": "f3ae3d8dc2ee9c74ab784205295b0388",
+      "passwordHash": "f67a4b77a6a4226cac15b8f2d08930768532549a3d2fb6985034c29814ea9ce497c08bbc02b657866f5dcca6646c7d0ba675b706157f05c476d283b636c8a94b"
+    },
+    {
+      "roles": [
+        "xds"
+      ],
+      "clientID": "lnsp",
+      "name": "lnsp",
```
Comment (HerbertYiga) [inline configs/openhim-core/initial-config.json:95]: added the lnsp cc @pmanko

**Review by pmanko — APPROVED** [review]

**Pull request merged**